hat/assets/js/apps/Iaso/domains/orgUnits/hooks/requests/useGetGroups.ts

```diff
@@ -55,3 +55,48 @@ export const useGetGroups = ({
         },
     });
 };
+
+// Correspondance between the name in the filter object and what the API expect
+const queryParamsMap = new Map([
+    ['dataSourceId', 'dataSource'],
+    ['sourceVersionId', 'sourceVersionId'],
+    ['blockOfCountries', 'blockOfCountries'],
+    ['appId', 'app_id'],
+]);
+
+type Params = {
+    dataSourceId?: number;
+    sourceVersionId?: number;
+    blockOfCountries?: string;
+    appId?: string;
+};
+export const useGetGroupDropdown = (
+    params: Params,
+): UseQueryResult<DropdownOptions<string>[], Error> => {
+    const queryParams = {};
+    queryParamsMap.forEach((keyInApi, keyInJS) => {
+        if (params[keyInJS]) {
+            queryParams[keyInApi] = params[keyInJS];
+        }
+    });
+    const urlSearchParams = new URLSearchParams(queryParams);
+    const queryString = urlSearchParams.toString();
+    return useSnackQuery({
+        queryKey: ['groups', queryString],
+        queryFn: () => getRequest(`/api/groups/dropdown/?${queryString}`),
+        snackErrorMsg: MESSAGES.fetchGroupsError,
+        options: {
+            staleTime,
+            select: data => {
+                if (!data) return [];
+                return data.groups.map(group => {
```

Comment:
There is no groups in data, you should map directly data `TypeError: data.groups is undefined.`
![Screenshot from 2023-05-15 10-04-54](https://github.com/BLSQ/iaso/assets/19631540/be432554-5a5c-4d85-9548-f4eca55b347f)